api/supabase/queries/volunteers.ts

```diff
@@ -0,0 +1,28 @@
+import { UUID } from 'crypto';
+import supabase from '../createClient';
+
+export async function fetchPerformer(event_id: UUID) {
```

Comment:
make this function name a bit more descriptive! something like `fetchPerformerByEventID`